specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-10-01/storagecache.json

```
@@ -316,7 +316,7 @@
             "type": "string"
           },
           {
-            "description": "Name of Cache. Length of name must be not greater than 80 and chars must be in list of [-0-9a-zA-Z_] char class.",
+            "description": "Name of Cache. Length of name must not be greater than 80 and chars must be in list of [-0-9a-zA-Z_] char class.",
```

```suggestion
            "description": "Name of Cache. Length of name must not be greater than 80 and chars must be from the [-0-9a-zA-Z_] char class.",
```

Suggestion applies to all instances of "must be in list of [...". (~14 instances)

---

Fixed in next update.